ceph_deploy/install.py

```diff
@@ -599,6 +600,12 @@ def make(parser):
                 (defaults to ceph.com)'
     )
 
+    repo.add_argument(
+        '--no-check-packages-signatures',
+        action='store_true',
+        help='do not check the package signatures',
+    )
+
```

Comment:
I don't think adding this arg to `repo` is what you want.  `repo` is a mutex group between `--adjust-repos | --no-adjust-repos | --repo`.  This would add the flag into that group.  Since this is seemingly relevant to all flags under `ceph-deploy install ...`, I think it would be `parser.add_argument()` instead.